playbooks/aws/aws_sdwan_config.yml

```diff
@@ -43,6 +43,12 @@ vmanage_instances:
   - hostname: "{{ aws_resources_prefix }}-vManage"
     system_ip: 192.168.1.1
     site_id: 111
+#  - hostname: "{{ aws_resources_prefix }}-vManage2"
```

Comment:
Add small comment that these are machines that should be uncommented if cluster setup.

Reply:
done